Reformats this repository into a jb book (#2)
ARCTraining/swd5_sc_py: CONTRIBUTING.md

> # Contributing to the template jupyter book docs repository

Reviewer: ```suggestion
# Contributing to the SWD5: Scientific Python repository
```

> :tada: **Welcome to the template jupyter book docs GitHub repository!** :tada:

Reviewer: ```suggestion
:tada: **Welcome to the SWD5: Scientific Python GitHub repository!** :tada:
```

_Thank you for wanting to contribute to this project!_

We hope that this guideline document will make it as easy as possible for you to get involved.

We welcome all contributions to this project via GitHub issues and pull requests.
Please follow these guidelines to make sure your contributions can be easily integrated in the project.

If you have any questions that aren't discussed below, please let us know through one of the many ways to [get in touch](#get-in-touch).

## Pull Request Process

1. Ensure the site builds fully on your local system within the conda environment provided
2. Your pull request will be test built locally by one of the maintainers
3. Any Continuous Integration steps must complete successfully
4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
   do not have permission to do that, you may request the second reviewer to merge it for you.

## Get in touch

You can get in touch with the Research Computing Team at Leeds via the following channels:

> - Open an [issue](https://github.com/ARCTraining/template-jb-docs/issues)

Reviewer: ```suggestion
- Open an [issue](https://github.com/ARCTraining/swd5_sc_py/issues)
```

> - Email the Research Computing Team via [ask-rc@leeds.ac.uk](mailto:ask-rc@leeds.ac.uk)

Reviewer: Just a general question (not PR related): Who looks at this email?

Author: This email is dead! I will remove
```suggestion
```
